process/consensus.md

```diff
@@ -9,24 +9,96 @@ Consensus is critical in a standards process:
+   Their title should identify them as editorial. Closing or merging them after
+   a full work day of positive review by at least one contributor of different
+   affiliation is acceptable as long as there is no objection. Editors shall
+   revisit the editorial nature of the change if there is any contention.
+
+2. Small technical changes or additions can be discussed in GitHub issues and
+   pull requests. It is the author's responsibility to ensure that interested
+   parties sign off. Simply mentioning people on GitHub may go unnoticed, the
+   onus remains on the author to contact interested parties and obtain their
+   feedback. Closing or merging after a full week of positive review by at least
+   three contributors of different affiliations is acceptable as long as there
+   is no objection. Consensus will be deemed to not have been reached if
+   interested parties did not sign off. At any point in time a contributor can
+   request that final consensus be delayed to a CG meeting. In this case, the
+   chair puts the item on the group's agenda of upcoming discussions.
+
+3. Substantial technical changes or additions are developed in their own GitHub
+   repository by a champion. It is critical that these proposals be able to
+   evolve quickly without much process. Early on in such a proposal's lifetime
+   no consensus is needed and a single champion can modify the proposal at will.
+   As the proposal matures it is expected that the champion will seek
+   collaborators to form a small group or
+   [subgroup](https://github.com/WebAssembly/meetings/blob/main/process/subgroups.md).
+   Gauging consensus within small groups is left up to the champion and gauging
+   consensus within subgroups is left up to the subgroup chair.
+
+Only 1. and 2. apply to the Working Group since the Community Group is the sole
+venue where substantial work can occur. It is expected that Working Group
+changes or additions in 2.'s category are validated by the Community Group
+before being adopted by the Working Group. It is expected that all changes or
+additions which reach consensus in the Community Group will be forwarded to the
+Working Group for adoption.
+
+When a proposal is near maturity the champion shall bring it to a CG or subgroup
+meeting and seek formal consensus on open design questions and contentious
+issues. All decisions made by the small group can be revisited until formal
+consensus is reached at a subgroup or CG meeting.
+
+When a proposal is ready for phase advancement, the champion shall bring it to a
+Community Group meeting and seek formal consensus for phase advancement. No
+proposal can advance phases without formal consensus in a CG meeting.
+
+## Formal consensus at meetings
+
+For in-person and online meetings, champions are expected to list points for
+which they will seek formal consensus in the meeting agenda at least 24 hours
+before the meeting[^phase-1-exception]. Only at in-person meetings can new
```

Comment:
I think I get what this is saying and it makes sense (in-person meetings have more time for discussion, fuller quorum, schedule slack we can fill, etc). But also it sometimes happens even at online meetings that the consensus point is modified from the original form based on agreement in the meeting. Not sure it's worth elaborating too much on that here though.,

Reply:
Yeah, I think it's ok if there is some flexibility in practice. The documentation here is enough that someone who misses an important vote because it was not properly scheduled can reasonably ask for a redo, which I think is WAI.